pkg/arvo/sys/hoon.hoon

```diff
@@ -10167,8 +10167,12 @@
   ++  mine
     ::  mint all chapters and feet in a core
     ::
-    |=  [mel/vair nym/(unit term) hud/poly dom/(map term tome)]
+    |=  [gol/type mel/vair nym/(unit term) hud/poly dom/(map term tome)]
     ^-  (pair type nock)
+    |^
+    =/  gol  (core-check gol)
+    =.  gol  (chapters-check gol)
```

Comment:
I don't like the shadowing here, seems likely to be confusing (in an already dense corner of the compiler). Maybe `gol/type` and `log/gol-type` or somesuch.